src/pages/Product/Product.js

```diff
@@ -1,33 +1,101 @@
-import './Product.css'
-import React from 'react'
-import ProductCSS from './Product.module.css'
-import { Filter, ProductCard } from '../../components'
-import { CartState } from '../../context/Context'
-
+import "./Product.css";
+import React from "react";
+import ProductCSS from "./Product.module.css";
+import { Filter, ProductCard } from "../../components";
+import { CartState } from "../../context/Context";
 
 export const Product = () => {
-  const {state: {products}} = CartState();
-  console.log(products)
+  const {
+    state: { products },
+    productState: {
+      sortBy,
+      rating,
+      sliderValue,
+      searchQuery,
+      categorySelfHelp,
+      categoryFiction,
+      categoryNonFiction,
+    },
+  } = CartState();
+  const transformProducts = () => {
+    let sortedProducts = products;
+    
+    // if (sortBy === null) {
```

Comment:
Please remove all commented code. It unnecessarily increased Lines of code(LOC) and do not look good.